db/sql.h

```diff
@@ -52,6 +52,12 @@ enum transaction_level {
     TRANLEVEL_SNAPISOL = 12
 };
 
+enum api_type {
+    API_TYPE_UNDEFINED = 1,
+    API_TYPE_CDB2API = 2,
+    API_TYPE_COMDB2API = 3
+};
```

Comment:
I don't love having this enum because it exposes non-open-source api information to the open-source code. Alternatively each plugin could just return an opaque string, then the reqlog would blindly print this string. We wouldn't be able to add any logic on top of that (logic to do something only if the api is a specific type), but it would keep comdb2api hidden from the open-source codebase. Thoughts?

Reply:
String sounds better. I doubt we would have any logic on top of it, and if we do, it'll most likely be implemented as another self-contained plugin callback.